src/opentracing/span.js

```diff
@@ -99,19 +100,20 @@ class SignalFxSpan extends Span {
 
   _addTags (keyValuePairs) {
     try {
-      Object.keys(keyValuePairs).forEach(key => {
-        this._spanContext._tags[key] = keyValuePairs[key]
-      })
+      Object.assign(
+        this._spanContext._tags,
+        truncate(keyValuePairs, this.tracer()._recordedValueMaxLength)
+      )
     } catch (e) {
       log.error(e)
     }
   }
 
   _log (keyValuePairs, timestamp) {
-    const logged = {
+    const logged = truncate({
```

Comment:
`_truncate`?

Reply:
I think this `truncate` is fine without a leading underscore, because it's our, as opposed to those prefixed instances which come from lodash.